framework/src/test/java/org/apache/felix/framework/BundleWiringImplTest.java

```diff
@@ -408,23 +380,23 @@ public void testFirstGeneratedAccessorSkipClassloading() throws Exception
         when(bundleWiring.getBundle().getFramework()).thenReturn(mockFramework);
 
         BundleClassLoader bundleClassLoader = createBundleClassLoader(
-                BundleClassLoader.class, bundleWiring);
-        assertNotNull(bundleClassLoader);
+                BundleClassLoader.class, bundleWiring);
+        assertThat(bundleClassLoader).isNotNull();
 
         try {
-            bundleClassLoader.loadClass(classToBeLoaded, true);
-            fail();
+            bundleClassLoader.loadClass(classToBeLoaded, true);
+            fail("");

```

Comment:
Add a message. Empty String is unhelpful.
```suggestion
            fail("Class should not load");

```